angular.json

```diff
@@ -106,6 +106,14 @@
               ],
               "serviceWorker": true,
               "ngswConfigPath": "ngsw-config.json"
+            },
+            "staging": {
```

Comment:
adding other properties here like in production would be good imo. Without other properties we can share source maps etc. However this is still internal env. So it is up to you.